src/_data/architecture/iap.yml

```diff
@@ -137,3 +137,10 @@ vendors:
     pricing: "Published"
     notes: ""
 
+  - company: "Akamai"
+    product: "Akamai Enterprise Application Access"
+    url: "akamai.com/eaa"
```

Comment:
I notice this URL is missing the protocol (https://) and redirects to https://www.akamai.com/products/enterprise-application-access, is there an reason not to use the target URL instead?

Reply:
Apologies, this was an oversight (updated).